dockstore-webservice/src/main/java/io/dockstore/webservice/core/Image.java

```diff
@@ -77,6 +77,14 @@
     @ApiModelProperty(value = "Stores the OS and, if available the OS version. Separated by a / and only applicable to Docker Hub", position = 7)
     private String os;
 
+    @Column()
+    @ApiModelProperty(value = "The size of the image in bytes")
+    private Long size;
+
+    @Column()
+    @ApiModelProperty(value = "The date the image was updated (not to be confused with when it's updated in Dockstore)")
```

Comment:
I find the "not to be confused" portion confusing. :) How about "The date the image was updated in the Docker repository"?